app/views/clients/_aliases.haml

```diff
@@ -1,10 +1,13 @@
 
 - aliases = @client.client_names(window: false, user: current_user).uniq
+- health_ds = GrdaWarehouse::DataSource.health.first&.short_name || 'Health'
+- client_has_health = aliases.detect { |name| name[:ds] == health_ds }
```

Comment:
This should probably detect *any* health DS instead of just the first.  Potentially there are multiple.  Also, let's push this logic off to the Client model.  (It can probably be rolled into `@client.add_patient_to_aliases`